pennylane/tape/qnode.py

```diff
@@ -140,13 +140,16 @@ def __init__(self, func, device, interface="autograd", diff_method="best", **dif
             )
 
         self.func = func
-        self.device = device
+        self.original_device = device

```

Comment:
```suggestion
        self._original_device = device
```
Is this one for users to access?